### PR TITLE
Brush size

### PR DESCRIPTION
**Pull request by ntnmrndn**

### CODE REVIEW

**Comment by muukii** [comment]
It looks good.

Could you put the screenshot that displays preview indicator? I’d like to see that UI.

Thanks.

**Comment by ntnmrndn** [comment]
IMG_152F88C6E3FA-1
Do you want me to add it to the readme ?

**Comment by muukii** [comment]
Thanks

I don’t think Readme needs it. For now.

By the way, Could you use custom slider that is used on other control panel?

**Comment by ntnmrndn** [comment]
@muukii I did. The API did'nt really cover that use case so the result is a bit awkward. works fine tho.

File: Sources/PixelEditor/Components/StepSlider.swift [hunk 1]
```diff
@@ -36,6 +36,11 @@ public final class StepSlider : UIControl {
       setupValues()
     }
   }
+  public var displayNumericValue = true {
+    didSet {
+      internalSlider.stepLabel.isHidden = !displayNumericValue
+    }
+  }
```
Comment (muukii) [inline Sources/PixelEditor/Components/StepSlider.swift]: I'd like to make it like this.

- Align naming
- Use computed property (avoid to manage the flag as several places. `internalSlider.stepLabel` be a source of truth)

```suggestion
  public var isStepLabelHidden: Bool {
    get { internalSlider.stepLabel.isHidden }
    set { internalSlider.stepLabel.isHidden = newValue }
  }
```

And, this suggestion will cause compile-error on caller.

**Comment by muukii** [comment]
> @muukii I did. The API did'nt really cover that use case so the result is a bit awkward. works fine tho.

Thanks, and sorry about that Custom slider 's API is not good. (I forgot that)
I think that slider should be improved with another PR in the future.

**Comment by muukii** [comment]
@ntnmrndn thx!
LGTM!

**Pull request merged**